io-functions-pay-portal/GetActivationStatus/__tests__/handler.test.ts

```diff
@@ -1,11 +1,92 @@
 /* tslint:disable: no-any */
 
-import { HttpHandler } from "../handler";
-
-describe("HttpCtrl", () => {
-  it("should return a string when the query parameter is provided", async () => {
-    const httpHandler = HttpHandler();
-    const response = await httpHandler();
-    expect(response.kind).toBe("IResponseSuccessJson");
-  });
+import { GetActivationStatusHandler } from "../handler";
+
+// should be HERE !!!!
```

Comment:
This comment is not so clear to me. I can't see _why_  `process.env` manipulation should be there. Maybe we should evaluate the use of a configuration file.

Reply:
you are right, I haven't investigated so much, the problem is that is used into `getConfigOrThrow` by `pagopa.ts` client. one more thing to review, tnx 👍